aten/src/THCUNN/generic/SpatialUpSamplingBilinear.cu

```diff
@@ -93,14 +97,14 @@ void THNN_(SpatialUpSamplingBilinear_updateGradInput)(
   int height2 = data2.getSize(2);
   int width2 = data2.getSize(3);
   assert(height1 > 0 && width1 > 0 && height2 > 0 && width2 > 0);
-  const accreal rheight= (height2 > 1) ? (accreal)(height1 - 1)/(height2 - 1) : accreal(0);
-  const accreal rwidth = (width2 > 1) ? (accreal)(width1 - 1) / (width2 - 1) : accreal(0);
+  const accreal rheight = linear_upsampling_compute_scale<accreal>(inputHeight, outputHeight, align_corners);
```

Comment:
Why is this inputHeight/outputHeight and not height1/height2?

Reply:
Oh I actually didn't notice. But these are equivalent. I will remove the `height/width*` ones as we already get the same numbers as `inputHeight etc` as input